### ISSUE TITLE
use_2to3 is invalid (setoptconf)

### ISSUE THREAD

**Issue by mithunsl** [pypa/setuptools#2782]
### setuptools version

setuptools==58.0.2

### Python version

3.6.13

### OS

Ubuntu 20.04

### Description

pip install setoptconf(https://pypi.org/project/setoptconf/) using setuptools 58.0.2 and above.

### Expected behavior

pip install setoptconf(or any other packages) should succeed.

### How to Reproduce

With setuptools 58.0.2, pip installing setoptconf(this a dependency from prospector) fails.

Falling back to 58.0.1, Install succeeds.

I understand that [2770](https://github.com/pypa/setuptools/pull/2770) specifically makes usage of use_2to3 invalid.

However, this seems to be breaking installation of widely used python packages. 

### Output

```
$ pip install setuptools==58.0.2
Collecting setuptools==58.0.2
  Using cached setuptools-58.0.2-py3-none-any.whl (816 kB)
Installing collected packages: setuptools
  Attempting uninstall: setuptools
    Found existing installation: setuptools 58.0.3
    Uninstalling setuptools-58.0.3:
      Successfully uninstalled setuptools-58.0.3
Successfully installed setuptools-58.0.2

$ pip install setoptconf
Looking in indexes: https://pypi.org/simple
Collecting setoptconf
  Using cached setoptconf-0.2.0.tar.gz (13 kB)
    ERROR: Command errored out with exit status 1:
     command: /src/.venv/bin/python3.6m -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-8xus1jkn/setoptconf_d71798b19c724112834cd40ba692ea0d/setup.py'"'"'; __file__='"'"'/tmp/pip-install-8xus1jkn/setoptconf_d71798b19c724112834cd40ba692ea0d/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-sf2rlgyz
         cwd: /tmp/pip-install-8xus1jkn/setoptconf_d71798b19c724112834cd40ba692ea0d/
    Complete output (1 lines):
    error in setoptconf setup command: use_2to3 is invalid.
    ----------------------------------------
    
    $ pip install setuptools==58.0.1
Looking in indexes: https://pypi.org/simple
Collecting setuptools==58.0.1
  Using cached setuptools-58.0.1-py3-none-any.whl (816 kB)
Installing collected packages: setuptools
  Attempting uninstall: setuptools
    Found existing installation: setuptools 58.0.2
    Uninstalling setuptools-58.0.2:
      Successfully uninstalled setuptools-58.0.2
Successfully installed setuptools-58.0.1

$ pip install setoptconf
Looking in indexes: https://pypi.org/simple
Collecting setoptconf
  Using cached setoptconf-0.2.0.tar.gz (13 kB)
Building wheels for collected packages: setoptconf
  Building wheel for setoptconf (setup.py) ... done
  Created wheel for setoptconf: filename=setoptconf-0.2.0-py3-none-any.whl size=13572 sha256=14decd564e54c20be9794ce6e669e1d3d94629f41bf3cce3776a8a56ed0a9d2a
  Stored in directory: /home/####/.cache/pip/wheels/ce/f2/8b/f7a4f5fb538575ddfae17603b9055a3ba74b351598b7d7a68a
Successfully built setoptconf
Installing collected packages: setoptconf
Successfully installed setoptconf-0.2.0
```

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

**Comment by haoyuanli** [comment]
I think setuptools>=58.0.0 is not supposed to install those aforementioned packages successfully, but they were on 58.0.1 and 58.0.2, so it's probably an expected behaviour

**Comment by haoyuanli** [comment]
`use_2to3` has been removed, relevant bug was fixed in 0.3.0, can you try `setoptconf==0.3.0` instead?

**Comment by mithunsl** [comment]
Yes. I have moved to use `setoptconf==0.3.0`. Thank you very much.
